packages/components/src/templates/map-marker-with-image.stories.ts

```diff
@@ -0,0 +1,30 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Map Marker with Image',
+  parameters: {
+    chromatic: { disableSnapshot: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=2544-19514&node-type=CANVAS&m=dev'
+    }
+  }
+};
+
+/**
+ * The default version of a `quote` pattern showcases a quote in an appealing layout,
```

Comment:
Delete description pls

Reply:
When this is done ... Approved from design side 🥳

Reply:
Updated